frontend/src/herdForm.tsx

```diff
@@ -35,6 +35,7 @@ import {
   FieldWithPermission,
   LimitedInputType,
 } from "@app/field_with_permission";
+import { getTarget } from "vis-util";
```

Comment:
Do not know why this is here

Reply:
I was also a bit confused. Sometimes VSCode is "smart" and auto-imports things that you accidentally write in the code. I've also had some of these unwanted imports causing bugs earlier.

Reply:
I  have removed it